solidity/test/Converter.js

```diff
@@ -220,7 +186,7 @@ describe('Converter', () => {
         reserveToken2 = await TestNonStandardToken.new('ERC Token 2', 'ERC2', 18, 2000000000);
     });
 
-    for (const type of [0, 1, 3, 4]) {
+    for (const type of [1, 3, 4]) {
```

Comment:
Let's all play along with that `2` doesn't exist 😅

Reply:
How exactly do you wanna handle this?

Reply:
I was joking. Nothing to do here 😎